src/foam/core/reflow/dashboard/TimeUnit.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * @license
+ * Copyright 2025 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.ENUM({
+  package: 'foam.core.reflow.dashboard',
+  name: 'TimeUnit',
```

Comment:
Foam already has a TimeUnit enum, can you just refine that instead?

Reply:
Its completely different  though, and its for the dashboard specifically?

Reply:
Yeah but why do we need two timeUnit enums, if you refine this one, the changes wont be deployed unless you use the dashboard code. The other one has the exact same values too, you are just adding on properties